docs/Acceptable-Formulae.md

```diff
@@ -21,11 +21,11 @@ We now accept versioned formulae as long as they [meet the requirements](Version
 
 ### We don’t like tools that upgrade themselves
 
-Software that can upgrade itself does not integrate well with Homebrew's own upgrade functionality. The self-update functionality should be disabled (while minimising complication to the formula).
+Software that can upgrade itself does not integrate well with Homebrew formulae's own upgrade functionality. The self-update functionality should be disabled (while minimising complication to the formula). It's fine (and well-supported) for Casks.
```

Comment:
Given this is a document about formulae, the cask route should probably point to where they need to go next, e.g.

```suggestion
Software that can upgrade itself does not integrate well with Homebrew formulae's own upgrade functionality. The self-update functionality should be disabled (while minimising complication to the formula). It's fine (and well-supported) for [Casks](Acceptable-Casks.md).
```

Reply:
I've been capitalizing "cask" when it's a proper noun, i.e. "Homebrew Cask" or the Cask class.